R/metab.bookkeep.R

```diff
@@ -64,7 +64,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
   
 	if("datetime"%in%names(mb.args)){ # check to see if datetime is in the ... args
 		datetime <- mb.args$datetime # extract datetime
-		freq <- calc.freq(datetime) # calculate sampling frequency from datetime
+		freq <- calc.freq(datetime) - 1 # calculate sampling frequency from datetime
 		if(nobs!=freq){ # nobs and freq should agree, if they don't issue a warning
```

Comment:
I think it should still be `if(nobs!=freq)`. For example, for hourly data and one midnight observation, there should be 24 `nobs` and 24 `freq`, thus a warning should appear if there's any missing data or two midnight observations (or other additional observations not during hourly time points).